providers/papermill/src/airflow/providers/papermill/operators/papermill.py

```diff
@@ -59,8 +59,6 @@ class PapermillOperator(BaseOperator):
         (ignores kernel name in the notebook document metadata)
     """
 
-    supports_lineage = True
```

Comment:
I think this removal should happen when 2.X gets out of scope - even though I feel it's not used at all

Reply:
Oh you’re right, this should be kept working on 2. I’ll add it back.

Reply:
If this was experimental feature we can just remove it but if we do lets just place a note in the affected providers changelog. I will rearrange the note to the proper place during release